### DISCUSSION TITLE
meta data dictionary

### DISCUSSION THREAD

**Discussion by namelus** [osm-search/Nominatim#2574] [Ideas]
Greetings @lonvia 

I'm wondering if there are any dictionaries of the data we can subscribe to?
Specifically, I just started looking at the Address Lookup instead of the Reverse API (as the docs recommed) and there is an attribute called OSM Id that is required. But I cannot find any good information about how to resolve this Id from Lat, Longs.

I have some thoughts about how to improve the documentation or at least what I'd like to see with it. If you're interested in hearing them let me know, and we can coordinate a meeting.

Thanks in advance for any direction/guidance.

Kind regards,

**Comment by mtmail** [comment]
The /reverse endpoint isn't deprecated. Only calling it with an `osm_type` and `osm_id` parameter is deprecated. https://nominatim.org/release-docs/latest/api/Reverse/

By 'meta data dictionaries' are you talking about OpenAPI/Swagger API specifications (https://github.com/osm-search/Nominatim/issues/1697)? Or are you looking for a list of places that exist in the database?

**Comment by lonvia** [comment]
Nominatim uses [OpenStreetMap](https://openstreetmap.org/abount) as its primary data source. You might want to read up on the project to better understand some of the documentation of Nominatim.

**Comment by namelus** [comment]
On Tue, Jan 11, 2022 at 6:16 PM mtmail ***@***.***> wrote:

> The /reverse endpoint isn't deprecated. Only calling it with an osm_type
> and osm_id parameter is deprecated.
> https://nominatim.org/release-docs/latest/api/Reverse/
>

Thanks for clarifying.

By 'meta data dictionaries' are you talking about OpenAPI/Swagger API
> specifications (#1697
> <https://github.com/osm-search/Nominatim/issues/1697>)? Or are you
> looking for a list of places that exist in the database?
>

 I was hoping to enhance the documentation. I visited the attached issue
you referenced, and that's what I'm talking about. I'll join that
discussion.

> —
> Reply to this email directly, view it on GitHub
> <https://github.com/osm-search/Nominatim/discussions/2574#discussioncomment-1951195>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACVRZWC3XPZQEYVXMT2RLBTUVS23JANCNFSM5LWWU5PA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Suleman*
ᐧ
